include/Gaffer/ValuePlug.h

```diff
@@ -205,6 +205,14 @@ class GAFFER_API ValuePlug : public Plug
 
 		//@}
 
+		/// Returns the current dirty count of this plug.
```

Comment:
I think we need to say what dirty count means, and touch on why we use it for capsules rather than the more usual hash. How about something like this?

```
/// Returns the number of times this plug has been dirtied.
/// This is used internally for cache invalidation but may also
/// be useful for debugging and as part of a "poor man's hash" where
/// computing the full upstream hash might be prohibitively
/// expensive (see `Encapsulate::hashObject()` for example).
```

Reply:
If we're describing this ... do we need to mention how it's affected by the dirty epoch?

Reply:
Ugh, hadn't thought of that, and that'll definitely make debugging a bit more confusing. How do we describe this so it's meaningful without getting into too much detail on the internals? "Returns a value that increments each time the plug is dirtied (but doesn't necessarily start at 0)"?